tests/test_dynamic_affect.py

```
@@ -0,0 +1,962 @@
+
+# =============================================================================
+# 12. EDGE CASES TESTS
+# =============================================================================
+
+class TestEdgeCases:
+    """Test edge cases and boundary conditions."""
+    
+    def test_all_zeros_input(self, tmp_path):
+        """Test processing all-zero affect input."""
+        system = DynamicAffectSystem(data_dir=tmp_path)
+        baseline = system.current.values.copy()
```

Ruff (0.15.0)

[error] Local variable `baseline` is assigned to but never used

Remove assignment to unused variable `baseline`

(F841)